indico/modules/events/abstracts/operations.py

```diff
@@ -62,3 +62,29 @@ def delete_abstract(abstract, delete_contrib=False):
     logger.info('Abstract %s deleted by %s', abstract, session.user)
     abstract.event_new.log(EventLogRealm.management, EventLogKind.negative, 'Abstracts',
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
+
+
+def schedule_cfa(event, data):
+    event.cfa.schedule(data)
+    logger.info("Call for abstracts for %s scheduled by %s", event, session.user)
+    date_format = '%d/%m/%Y %H:%M'
+    log_data = {
+        'Start': data['start_dt'].strftime(date_format),
```

Comment:
We use `format_date`:

* https://github.com/indico/indico/blob/master/indico/modules/events/timetable/operations.py#L85
* https://github.com/indico/indico/blob/master/indico/modules/events/timetable/operations.py#L111
* https://github.com/indico/indico/blob/master/indico/modules/events/timetable/operations.py#L123
* https://github.com/indico/indico/blob/master/indico/modules/events/timetable/reschedule.py#L97

Should I use `format_date` too, or just `.isoformat()`?